typings/index.d.ts

```diff
@@ -3189,6 +3189,8 @@ declare module 'discord.js' {
     maxUsers?: number;
   }
 
+  interface ReplyMessageOptions extends Omit<MessageOptions, 'replyTo'> {}
```

Comment:
```suggestion
  type ReplyMessageOptions = Omit<MessageOptions, 'replyTo'>;
```

Reply:
I'm quite new to TS. Will this change make any (substantial) difference or is it just a preference thing?

Reply:
it's the correct way of doing it but shouldn't make a difference afaik

Reply:
however, we might wanna keep the `interface` code since we'll be adding support for `fail_if_not_exists` which is only for replies, so the {} won't be empty anymore

Reply:
#5298 @ckohen added the new field for `fail_if_not_exist` in `MessageOptions`. Thus, `ReplyMessageOptions` will inherit that field too. So, should we accept this change now?

Reply:
This thing resolved itself with the latest commit 6e1b85f